cli/bin/stax_daemon.dart

```diff
@@ -2,7 +2,15 @@ import 'dart:async';
 import 'dart:io';
 
 void main(List<String> arguments) async {
-  final daemonPort = 5000;
+  int daemonPort = 62261;
+
+  for (int i = 0; i < arguments.length; i++) {
+    if (arguments[i] == '-p' && i + 1 < arguments.length) {
+      daemonPort = int.tryParse(arguments[i + 1]) ?? 5000;
```

Comment:
You can use flag for parsing https://github.com/TarasMazepa/stax/blob/main/cli/lib/command/flag.dart